.github/workflows/release.yml

```diff
@@ -0,0 +1,70 @@
+name: Build Releases
+
+on:
+  release:
+    types: [published]
+  push:
+    branches:
+      - 'master'
```

Comment:
How does it know to build the v1.3.0 or whatever tag in master?

Reply:
1. If the trigger is a push to master or develop, the tag will be the branch name
2. If the trigger is a release, the tag will be the tag name, which is equal to the version number